docs/datagrid-user-xml.adoc

```diff
@@ -0,0 +1,37 @@
+== Custom User Standalone XML Configuration
+= Usage
+
+To provide custom configurations you must:
+1. Create a config map containing at least `standalone.xml` which is mounted at `/opt/datagrid/standalone/configuration/user`
+2. Set the environment variable `USER_CONFIG_MAP=true` in your template
+3. Expose the required services and ports in your template
+
+When `USER_CONFIG_MAP=true` our image copies the contents of `opt/datagrid/standalone/configuration/user`. i.e. the configmap, to `/opt/datagrid/standalone/configuration`.
+This means that in addition to passing `standalone.xml`, it is also possible to provide `logging.properties`, `application-role.properties` etc in the configmap.
+
+Creating a configmap containing the desired configuration files is trivial, as an entire directory can be used to create a configmap consisting of filenames and their contents as key/values in the configmap. This can be achieved via `oc create configmap <name-of-configmap> --from-file=<local-dir>`.
+
+To create an application that uses a custom configuration, login to your openshift cluster via `oc`, push your built image to the cluster's repository and execute the following:
+
+1. `oc create -f docs/examples/user-configuration/user-config-template.yaml`
```

Comment:
Is `user-config-template.yaml` right? Seems like the file is called `user-xml-template.yaml` ?